root/oauth2/OAuth2Error.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+
+type Props = {|
+  +errorDescription: string,
+  +errorMessage: string,
+|};
+
+const Oauth2Error = ({
+  errorDescription,
+  errorMessage,
+}: Props) => (
+  <Layout fullWidth title={l('OAuth Authorization Error')}>
+    <h1>{texp.l('Error: {error}', {error: errorMessage})}</h1>
+    <p>{errorDescription}</p>
+    <p>{exp.l('{doc|Learn more}', {doc: '/doc/Development/OAuth2'})}</p>
+  </Layout>
+);
+
+export default Oauth2Error;
```

Comment:
File name `OAuth2Error` and default component name `Oauth2Error` are inconsistent about capitalization. `OAuth2` seems to be more frequently used.